src/ol/renderer/webgl/webglvectorlayerrenderer.js

```diff
@@ -58,6 +58,13 @@ ol.renderer.webgl.VectorLayer = function(mapRenderer, vectorLayer) {
    */
   this.replayGroup_ = null;
 
+  /**
+   * The last layer state.
+   * @private
+   * @type {?ol.layer.LayerState}
```

Comment:
In fact it is required, because `ol.layer.LayerState` is only a [`typedef`](https://github.com/openlayers/ol3/blob/master/src/ol/layer/layerbase.js#L29).